.github/workflows/otel-integration-e2e-test.yml

```diff
@@ -54,6 +54,7 @@ jobs:
           KUBECONFIG: ${{ env.KUBECONFIG }}
         run: |
           cd ./otel-integration/k8s-helm/e2e-test/
+          go clean -testcache
```

Comment:
Why this?

Reply:
Since it is a go test, it can happen that the result is being cached. It happened to me while working on this PR... I assume there should not be a problem for GitHub workflows but... I prefer to run this here to just double check cache doesn't play with our feelings.